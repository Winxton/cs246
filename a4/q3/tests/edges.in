new 2
init
0 0
1 0
1 1
-1 -1
print
step
print
step 100
print
