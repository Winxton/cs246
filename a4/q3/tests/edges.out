X_
XX
XX
XX
XX
XX
